distribution/lib/Standard/Base/0.0.0-dev/src/Errors/Common.enso

```diff
@@ -553,3 +554,27 @@ type Out_Of_Range
     to_text self =
         extra = if self.message.is_nothing then "" else ": "+self.message.to_text
         "(Out_Of_Range (value = "+self.value.to_text+")" + extra + ")"
+
+## Indiciates that the response from a remote endpoint is over the size limit.
+type Response_Too_Large
+    ## PRIVATE
+    Error size:Integer limit:Integer url:(Text|Nothing)
+
+    ## PRIVATE
+       Create a human-readable version of the error.
+    to_display_text : Text
+    to_display_text self =
+        url_maybe = if self.url.is_nothing then "" else "(URL "+self.url+")"
+        suggestion = "Use `Data.fetch` with `cache_policy=Dont_Cache_Responses`, or use `Data.download` to fetch the data to a local file, and `Data.read` to read the file."
+        "Response too large: repsonse size ("+self.size.to_text+") is over the limit ("+self.limit.to_text+")" + url_maybe + suggestion
```

Comment:
I think there's no space between the last `)` and the `Use ...` inside of `suggestion`.
```suggestion
        suggestion = " Use `Data.fetch` with `cache_policy=Dont_Cache_Responses`, or use `Data.download` to fetch the data to a local file, and `Data.read` to read the file."
        "Response too large: repsonse size ("+self.size.to_text+") is over the limit ("+self.limit.to_text+")" + url_maybe + suggestion
```

Reply:
Done